### PR TITLE
Add tabbar navigation

### PR DESCRIPTION
**Pull request by DejanuTeodora**
## Description
Added tab bar navigation

## Related
https://trello.com/c/Dz5zbLDN

## Documentation
https://www.hackingwithswift.com/quick-start/swiftui/adding-tabview-and-tabitem

## Steps to Test or Reproduce

1. Install app
2. Open app
3. Tap one of the tab bar elements

### CODE REVIEW

**Comment by DejanuTeodora** [comment]
[iOS - Add tabbar navigation](https://trello.com/c/Dz5zbLDN/160-ios-add-tabbar-navigation)

**Review by vladstanescu94 — APPROVED** [review]

File: Shared/Features/Freaks/View/HomeView.swift [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import SwiftUI
+
+struct HomeView: View {
```
Comment (vladstanescu94) [inline Shared/Features/Freaks/View/HomeView.swift:3]: We need to move this file to another directory.

File: Shared/Features/Freaks/View/MainView.swift [hunk 2]
```diff
@@ -0,0 +1,27 @@
+import SwiftUI
+
```
Comment (vladstanescu94) [inline Shared/Features/Freaks/View/MainView.swift:2]: We need to move this file to another directory.

File: Shared/Features/Freaks/View/ProjectsView.swift [hunk 3]
```diff
@@ -0,0 +1,13 @@
+import SwiftUI
+
```
Comment (vladstanescu94) [inline Shared/Features/Freaks/View/ProjectsView.swift:2]: We need to move this file to another directory.

**Review by vladstanescu94 — CHANGES_REQUESTED** [review]

**Review by MihaiPantiru — APPROVED** [review]

**Review by vladstanescu94 — APPROVED** [review]

**Comment by cipry13** [comment]
[Android - Freaks Screen - get and show freaks from API](https://trello.com/c/8YqKWs3B/154-android-freaks-screen-get-and-show-freaks-from-api)

**Pull request merged**